test/controllers/slides_controller_test.rb

```diff
@@ -143,4 +143,38 @@ class SlidesControllerTest < ActionController::TestCase
 
     assert_equal "horizontal", slide.reload.orientation
   end
+
+  test "assert that `controls muted` attribute is present on slides with foreground_type: video" do
+    # Arrange
+    super_admin = users(:super_admin)
```

Comment:
Thanks for pointing that out. I added that in when Tom mentioned to arrange to be an admin user and then forgot about it while trying to fix the rest of the test. I changed it to sign in as an admin instead.